src/charm.py

```diff
@@ -264,15 +266,15 @@ def _replica_consensus_reached(self):
             return False
         return all(replica_data.get(f) for f in fields)
 
-    def _on_leader_elected_replica_data_handler(self, event):
+    def _on_leader_elected_replica_data_handler(self, _event):
```

Comment:
Is there are reason why this is a named args instead of a throwaway arg`_`?

Reply:
It's a convention to mark a unused argument.

Reply:
I think both could work, I just prefer this way.

Reply:
I was just curious. Thx for the clarification